cmd/sonobuoy/app/args.go

```diff
@@ -87,6 +90,12 @@ func AddKubeconfigFlag(cfg *Kubeconfig, flags *pflag.FlagSet) {
 	flags.Var(cfg, "kubeconfig", "Path to explicit kubeconfig file.")
 }
 
+// AddPluginFlag describes which plugin's images to interact
+func AddPluginFlag(cfg *string, flags *pflag.FlagSet) {
+	// The default is the empty string (look in the environment)
+	flags.StringVarP(cfg, pluginFlag, "p", "", "Describe which plugin's images to interact.")
```

Comment:
nit: `interact with`

Reply:
I'd also list the valid values here on the flag since that will be shown when/if they dont provide it.